app/src/main/res/layout/activity_login.xml

```diff
@@ -124,116 +126,118 @@
             android:background="@drawable/btn_icon_google"
-            tools:ignore="SpeakableTextPresentCheck,SpeakableTextPresentCheck,SpeakableTextPresentCheck,SpeakableTextPresentCheck" />
+            tools:ignore="SpeakableTextPresentCheck,SpeakableTextPresentCheck,SpeakableTextPresentCheck,SpeakableTextPresentCheck"
+            style="?android:attr/buttonBarButtonStyle" />
     </LinearLayout>
 
     <TextView
         android:id="@+id/txt_noAccount"
-        android:layout_width="120dp"
-        android:layout_height="25dp"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/btns_apps"
-        android:text="@string/no_account"
-        android:textSize="15sp"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/dp14"
+        android:fontFamily="@font/roboto_light_italic"
         android:gravity="center"
-        app:layout_constraintHorizontal_bias="0.25"
+        android:text="@string/no_account"
         android:textColor="@color/tertiary"
-        android:fontFamily="@font/roboto_light_italic"
+        android:textSize="@dimen/sp15"
         android:textStyle="bold"
-        android:layout_marginTop="14dp"
-        />
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.25"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/btns_apps"
+        tools:ignore="TextContrastCheck" />
 
     <TextView
         android:id="@+id/txt_signUp"
-        android:layout_width="75dp"
-        android:layout_height="25dp"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
         android:fontFamily="@font/roboto_light_italic"
         android:gravity="center"
         android:text="@string/signUp"
         android:textColor="@color/secondary"
-        android:textSize="15sp"
+        android:textSize="@dimen/sp15"
         android:textStyle="bold"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHorizontal_bias="0"
         app:layout_constraintStart_toEndOf="@id/txt_noAccount"
         app:layout_constraintTop_toBottomOf="@id/btns_apps"
-        android:layout_marginTop="14dp"
+        android:layout_marginTop="@dimen/dp14"
         />
     <TextView
         android:id="@+id/txt_guest"
-        android:layout_width="200dp"
-        android:layout_height="25dp"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintTop_toBottomOf="@id/txt_noAccount"
         android:text="@string/guest"
         android:gravity="center"
-        android:textSize="16sp"
+        android:textSize="@dimen/main_activity_txt_size"
```

Comment:
remember not to tie dimens to specific screens when they can be reusable

Reply:
i'll implement it, thanks